src/pages/PostPage/PostPage.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useState } from 'react'
+import * as S from './PostPage.style.js';
+import Button from '../../components/MainButton';
+import SelectedImg from '../../assets/icon/background-selected.png'
+
+const PostPage = () => {
+  const [choiceColor, setChoiceColor] = useState(true);
+  const [choiceImage, setChoiceImage] = useState(false);
```

Comment:
```suggestion
    const [backgroundType, setBackgroundType] = useState('color');
```

이런식으로 타입을 분리하는건 어떨까요?
active되어 있는값은 항상 하나인데 상태를 여러개로 관리하는건 효율성 측면에서 리소스 낭비입니다!